src/exodus/rpctx.cpp

```diff
@@ -1481,6 +1481,53 @@ UniValue exodus_sendalert(const UniValue& params, bool fHelp)
     }
 }
 
+UniValue exodus_sendcreatedenomination(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() != 3) {
+        throw std::runtime_error(
+            "exodus_sendcreatedenomination \"fromaddress\" propertyid \"value\"\n"
+            "\nCreate a new denomination for the given property.\n"
+            "\nArguments:\n"
+            "1. fromaddress          (string, required) the address to send from\n"
+            "2. propertyid           (number, required) the property to create a new denomination\n"
+            "3. value                (string, required) the value of denomination to create\n"
```

Comment:
better to also support number.

Reply:
We can't due to it will have problems with JavaScript client.